### PR TITLE
updated the outbound call

### PR DESCRIPTION
**Pull request by Rani0608**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: ServerRecording/src/main/resources/config.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
-Connectionstring=endpoint=https://REDACTED.communication.azure.com/;accesskey=QWNjZXNzS2V5
-BlobStorageConnectionString=DefaultEndpointsProtocol=https;AccountName=REDACTED;AccountKey=QWNjZXNzS2V5;EndpointSuffix=core.windows.net
-ContainerName=acs-recording-container-name
\ No newline at end of file
+Connectionstring=endpoint=https://acs-app-validations.communication.azure.com/;accesskey=F7/3U887zz4Ur5CZwhQmGLml8y9uWVb6ir4kqI2K4FyGtShWFxdYBLBf4ZxtJ41P6C5KUC2sZpfayCOtl9DMbw==
```
Comment (moirf) [inline ServerRecording/src/main/resources/config.properties:1]: should be all blank

File: ServerRecording/src/main/java/com/acsrecording/api/Controller/CallRecordingController.java [hunk 2]
```diff
@@ -311,21 +339,19 @@ private void processFile(String url, String documentId, String fileType, String
         String filePath = String.format(".\\%s", fileName);
         Path path = Paths.get(filePath);
         logger.log(Level.INFO, String.format("Local file Path url -- > %s", filePath));
-
-        var downloadResponse = callAutomationClient.getCallRecording().downloadToWithResponse(url, path, null, null);
-        logger.log(Level.INFO, String.format("Download media response --> %s", getResponse(downloadResponse)));
-        logger.log(Level.INFO, String.format("Download media request --> %s", getRequestData(downloadResponse)));
+        this.callAutomationClient.getCallRecording().downloadTo(url,path,null,null);
+        //callAutomationClient.getCallRecording().downloadTo(url,path,null,null);
```
Comment (moirf) [inline ServerRecording/src/main/java/com/acsrecording/api/Controller/CallRecordingController.java:343]: remove all commented

File: ServerRecording/src/main/java/com/acsrecording/api/Controller/CallRecordingController.java [hunk 3]
```diff
@@ -50,22 +56,44 @@
...
     Logger logger;
     String recordingFileFormat;
     private final com.azure.communication.callautomation.CallAutomationClient callAutomationClient;
 
     CallRecordingController() {
         ConfigurationManager configurationManager = ConfigurationManager.getInstance();
         String connectionString = configurationManager.getAppSettings("Connectionstring");
+        ACSAcquiredPhoneNumber= configurationManager.getAppSettings("ACSAcquiredPhoneNumber");
         container = configurationManager.getAppSettings("ContainerName");
         blobStorageConnectionString = configurationManager.getAppSettings("BlobStorageConnectionString");
```
Comment (moirf) [inline ServerRecording/src/main/java/com/acsrecording/api/Controller/CallRecordingController.java:71]: remove all storage related codes.

File: ServerRecording/pom.xml [hunk 4]
```diff
@@ -74,12 +74,24 @@
 		</dependency>
 	</dependencies>
 	<repositories>
+		<repository>
```
Comment (moirf) [inline ServerRecording/pom.xml:77]: we should not add private repo path into public code

File: CallRecording/src/main/java/com/acsrecording/api/Models/VideoConfiguration.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
+package com.acsrecording.api.Models;
```
Comment (moirf) [inline CallRecording/src/main/java/com/acsrecording/api/Models/VideoConfiguration.java:1]: check if it really required

File: CallRecording/src/main/java/com/acsrecording/api/Models/Mapper.java [hunk 6]
```diff
@@ -0,0 +1,43 @@
+package com.acsrecording.api.Models;
```
Comment (moirf) [inline CallRecording/src/main/java/com/acsrecording/api/Models/Mapper.java:1]: check use of this, i belive not all required

File: ServerRecording/src/main/java/com/acsrecording/api/Controller/CallRecordingController.java (hunk 3 above)
Comment (Rani0608) [reply ServerRecording/src/main/java/com/acsrecording/api/Controller/CallRecordingController.java:71]: Created new folder as CallRecoeding For ServerRecording one..

**Comment by Rani0608** [comment]
creating new Branch and PR.

**Pull request closed without merging**